Formula/dosbox-staging.rb

```diff
@@ -48,8 +49,7 @@ class DosboxStaging < Formula
 
   def install
     (buildpath/"subprojects").rmtree # Ensure we don't use vendored dependencies
-    system_libs = %w[fluidsynth glib iir mt32emu opusfile png sdl2 sdl2_net slirp speexdsp zlib]
```

Comment:
Any commit or code reference about this chnage?

Reply:
The Meson option `system_libraries` was dropped in commit dosbox-staging/dosbox-staging@53bacf8. That option was used to determine `system_libs_list` in `meson.build`, and the list was only used in the `allow_fallback` options in the `dependency` declarations (see [diff](https://github.com/dosbox-staging/dosbox-staging/commit/53bacf8)).

Homebrew-provided deps will still be used after this change because: 1. `allow_fallback` will default to `false` since no dependencies have been declared with `required: true` (see Meson's [docs](https://mesonbuild.com/Reference-manual_functions.html#arguments17)); 2. line 51 removes the subprojects that vendor those deps.

Reply:
Sounds good 👍